Okta.AspNetCore/OktaMvcOptions.cs

```diff
@@ -70,5 +70,11 @@ public class OktaMvcOptions : AspNet.Abstractions.OktaWebOptions
         /// Gets or sets the event invoked if exceptions are thrown during request processing.
         /// </summary>
         public Func<AuthenticationFailedContext, Task> OnAuthenticationFailed { get; set; } = context => Task.CompletedTask;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to retrieve additional "long" claims for AspNet Identity from the UserInfo endpoint after login.
```

Comment:
I'm not sure what "long" means here; quoting it implies that it should be a known concept; providing a definition would be helpful, or even replacing "long" with the definition of what it means.

Reply:
@bryanapellanes-okta  It's about long claim names and their shorter equivalents (`http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier` instead of  `sub`). Can you help me to come up with a better concise and descriptive text?

Reply:
I would use something like
`Gets or sets a value indicating whether to use fully qualified identity claim names.`

I would event consider changing the property name to UseFullyQaulifiedIdentityClaimNames

Reply:
I think I did not explained it correctly, shorter claim names are used as before, but if this option is set, 5 claims with long names (fully qualified) will be added to claim actions array.
That's why I've made it active by default, it only adds something to the existing list.
So may be 'AddFullyQaulifiedIdentityClaimNames' ?

Reply:
👍 Yes, I like that property name.  The doc comment could be something like this then:

```
Gets or sets a value indicating whether to include identity claims with fully qualified claim names.
```